src/Common/Log/DoctrineHandler.php

```diff
@@ -16,6 +16,9 @@ class DoctrineHandler extends AbstractProcessingHandler
     private TokenStorageInterface $tokenStorage;
     private int $minLevel;
 
+    private const SECRET_VALUE = '***';
+    private const SECRET_KEYS = ['api_key'];
```

Comment:
Define in the constructor? no? As it might be specific for the core then the emsch?

Reply:
Would not do that for now. If we face it we can change it :)